examples/step-33/doc/results.dox

```diff
@@ -263,6 +258,23 @@ build the Newton matrix each time, the resulting scheme may well be
 faster.
 
 
+<h4>Cache the explicit part of residual</h4>
+
+The residual calulated in ConservationLaw::assemble_cell_term function
+read
+   $R_i = \left(\frac{\mathbf{w}^{k}_{n+1} - \mathbf{w}_n}{\delta t}
+    , \mathbf{z}_i \right)_K  +
+      \theta \mathbf{B}({\mathbf{w}^{k}_{n+1})(\mathbf{z}_i)_K +
+      (1-\theta) \mathbf{B}({\mathbf{w}_{n}) (\mathbf{z}_i)_K $
+means that we calculate the spacial residual twice at one Newton
```

Comment:
insert "This" or something similar at the start of this line. Also: spacial -> spatial